src/cli/list.rs

```diff
@@ -75,9 +75,13 @@ struct PackageToOutput {
 
 pub async fn execute(args: Args) -> miette::Result<()> {
     let project = Project::load_or_else_discover(args.manifest_path.as_deref())?;
-    let environment_name = args
-        .environment
-        .map_or_else(|| EnvironmentName::Default, EnvironmentName::Named);
+    let environment_name = args.environment.map_or_else(
```

Comment:
This can be achieved with clap directly: https://docs.rs/clap/latest/clap/struct.Arg.html#method.env

Reply:
Thanks! addressed..